src/main/java/io/gravitee/reporter/common/formatter/elasticsearch/ElasticsearchFormatter.java

```diff
@@ -446,6 +446,9 @@ private void addCommonFields(
       if (esOptions.get("pipeline") != null) {
         data.put("pipeline", esOptions.get("pipeline"));
       }
+      if (esOptions.get("date") == null) {
+        data.put("date", sdf.format(reportable.timestamp()));
+      }
```

Comment:
Is it a fix from your test ?

Reply:
Yes exactly, the date is mandatory and if you give a empty map as options it actually fails to build the template